monai/transforms/compose.py

```diff
@@ -136,3 +139,17 @@ def __call__(self, input_):
         for _transform in self.transforms:
             input_ = apply_transform(_transform, input_)
         return input_
+
+    def inverse(self, data, keys: Optional[Tuple[Hashable, ...]] = None):
+        if not isinstance(data, Mapping):
+            raise RuntimeError("Inverse method only available for dictionary transforms")
+        d = deepcopy(dict(data))
+        if keys:
+            keys = ensure_tuple(keys)
+
+        # loop backwards over transforms
+        for t in reversed(self.transforms):
+            # check if transform is one of the invertible ones
+            if isinstance(t, InvertibleTransform):
+                d = t.inverse(d, keys)
```

Comment:
will we have a case that we want to modify keys as well, `d, keys = t.inverse(d, keys)`?